forge-gui/res/cardsfolder/upcoming/kharn_the_betrayer.txt

```diff
@@ -0,0 +1,15 @@
+Name:Kharn the Betrayer
+ManaCost:3 R
+Types:Legendary Creature Astartes Berserker
+PT:5/1
+S:Mode$ MustAttack | ValidCreature$ Card.Self | Description$ Berzerker — CARDNAME attacks or blocks each combat if able.
+S:Mode$ MustBlock | ValidCreature$ Card.Self
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Any | ValidCard$ Card.Self | Execute$ TrigDraw1 | TriggerDescription$ Sigil of Corruption — When you lose control of CARDNAME, draw two cards.
+SVar:TrigDraw1:DB$ Draw | NumCards$ 2
+T:Mode$ ChangesController | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigDraw2 | Secondary$ True | TriggerDescription$ Sigil of Corruption — When you lose control of CARDNAME, draw two cards.
+SVar:TrigDraw2:DB$ Draw | NumCards$ 2 | Defined$ TriggeredOriginalController
```

Comment:
Another Draw should not not be needed, you need to control the trigger anyway
Try using `TriggerController$`